pages/source-code.html

```diff
@@ -0,0 +1,200 @@
+            <p>There are other repositories containing source code that we package and distribute in Uyuni, but it is used in other projects (so we don't have it at our <a href="https://github.com/uyuni-project" target="_blank">main GitHub project</a>:</p>
+            <ul>
+                <li><a href="https://github.com/openSUSE/salt" target="_blank">salt</a>: SUSE and openSUSE patches and backports for SaltStack.</li>
+                <li><a href="https://github.com/SUSE/salt-netapi-client" target="_blank">salt-netapi-client</a>: Java bindings for the Salt API.</li>
+                <li><a href="https://github.com/SUSE/salt-formulas" target="_blank">salt-formulas</a>: Salt Formulas for SUSE Enterprise Linux, openSUSE Linux and Uyuni.</li>
+                <li><a href="https://github.com/openSUSE/virtual-host-gatherer" target="_blank">virtual-host-gatherer</a>: Script to gather information about virtual system running on different kind of hypervisors.</li>
+                <li><a href="https://github.com/openSUSE/subscription-matcher" target="_blank">subscription-matcher</a>: Tool to report whether some installed SUSE products match a set of SUSE subscriptions.</li>
+                <li><a href="https://github.com/openSUSE/containment-rpm-pxe" target="_blank">containment-rpm-pxe</a>: Wrap PXE images built by KIWI on OBS into RPMs.</li>
+            </ul>
+            <h4><a href="#openbuildservice">OpenBuildService</a></h4>
+            <p>Besides or the Git repositories, there are certain packages we maintain directly at OpenBuildService.</p>
+            <p>There are two kind of reasons for this:</p>
+            <ul>
+                <li>Packages that contain only upstream code and (in some cases) some patches. It is the case for all packages at <a href="https://build.opensuse.org/project/show/systemsmanagement:Uyuni:Master:Other" target="_blank">systemsmanagement:Uyuni:Master:Other</a>.</li>
+                <li>
+                    The package does not really contain code. Some examples of this are:
+                    <ul>
+                        <li><a href="https://build.opensuse.org/package/show/systemsmanagement:Uyuni:Master/000product" target="_blank">000product</a> (the product definition for the Uyuni Server and the Uyuni Proxy).</li>
+                        <li><a href="https://build.opensuse.org/package/show/systemsmanagement:Uyuni:Master/000CloudImages-Server" target="_blank">000CloudImages-Server</a> (the cloud image definition for the Uyuni Server).</li>
+                        <li><a href="https://build.opensuse.org/package/show/systemsmanagement:Uyuni:Master/000CloudImages-Proxy" target="_blank">000CloudImages-Proxy</a> (the cloud image definition for the Uyuni Proxy).</li>
+                        <li><a href="https://build.opensuse.org/package/show/systemsmanagement:Uyuni:Master/patterns-uyuni" target="_blank">patterns-uyuniy</a> (the patterns for the Uyuni Server and the Uyuni Proxy).</li>
```

Comment:
I think there is a typo in the link name

Reply:
True :-D

Reply:
Fixed.